build_all/mbed5/build.sh

```diff
@@ -1,7 +1,11 @@
 # Copyright (c) Microsoft. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 sudo rm -R -f mxchip
-git clone -b rajeev/latest_mbed https://github.com/massand/mxchip_az3166_firmware.git mxchip
+
+git clone https://github.com/massand/mxchip_az3166_firmware.git mxchip
+cd mxchip
+git checkout $SOURCE_BRANCH
```

Comment:
>$SOURCE_BRANCH [](start = 13, length = 14)

Rename to something more specific like MXCHIP_REPO_SOURCE_BRANCH?

Reply:
Added.